frappe/integrations/frappe_providers/frappecloud.py

```diff
@@ -19,8 +18,12 @@ def frappecloud_migrator(local_site):
 		)
 		return
 
-	script_contents = request.json()["message"]
+	return request.json()["message"]
 
+
+def frappecloud_migrator(context):
```

Comment:
`context` isn't being used here, can drop it.

I assume all site choice logic is handled in the script downloaded from press now?

Reply:
yes.